lib/firebase.ts

```diff
@@ -39,4 +41,43 @@ export async function getMovieById(id : number) {
 
     // @ts-ignore
     return docToJSON((await getDocs(genreQuery)).docs[0]);
+}
+
+export async function addMovieToList(id: string, user: any, list: string, userInfo: any){
```

Comment:
This should likely be moved to an api route in the future, for server-side validation to be possible

Reply:
Once that'll be implemented, we'll make the change.